### PR TITLE
Adding privacy modal to dataset view

### PR DESCRIPTION
**Pull request by xavijam**
cc @nobuti

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/dataset/dataset-header/header.tpl [hunk 1]
```diff
@@ -15,9 +15,9 @@
     </button>
   </div>
   <div class="u-bSpace--xl">
-    <button class="u-rSpace--xl u-actionTextColor js-privacy">
+    <<%- privacyDOMElement %> class="u-rSpace--xl u-actionTextColor <% if (isOwner) { %>js-privacy<% } %>">
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/dataset/dataset-header/header.tpl]: For me it's clearer a simple if but I see the point.

**Comment by nobuti** [comment]
💯

**Comment by xavijam** [comment]
Thanks for making it so easier :kiss:

**Comment by xavijam** [comment]
Failing due to the same thing for sync-datast, a lint error. It will be fixed when sql thing is merged.

**Pull request merged**